recipes/emsdk_installer/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, tools
+import os
+
+
+class EmSDKInstallerConan(ConanFile):
+    name = "emsdk_installer"
+    description = "Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/bincrafters/conan-emsdk_installer"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("conan", "emsdk", "emscripten", "installer", "sdk")
+    license = "MIT"
+
+    short_paths = True
+    _source_subfolder = "source_subfolder"
+
+    def source(self):
```

Comment:
Remove `settings.os` in `package_info` to make sure that only one package is built with cross-os support.
```suggestion
    @property
    def _source_subfolder(self):
        return "source_subfolder"

    def package_id(self):
        del self.info.settings.os

    def source(self):
```